.travis.yml

```diff
@@ -26,7 +28,7 @@ before_install:
   - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-ce
 
 install:
-  - make .venv_ansible${ANSIBLE_INSTALL_VERSION}
+  - make -B .venv
```

Comment:
Are you using -B to ensure that the target always runs? You can use the PHONY directive for that in the Makefile.

Reply:
copy and paste error.

I will send separate PR to scaffold with cache folders + `-B` in install, and explain reasoning there.

For now I'm deleting -B :)